playground/pages/index.vue

```diff
@@ -1,8 +1,8 @@
-<script setup>
+<script setup lang="ts">
 const story = await useAsyncStoryblok("vue", {
   version: "draft",
   language: "en",
-  resolve_relations: ["popular-articles.articles"]
+  resolve_relations: "popular-articles.articles"
```

Comment:
This should be an array, any reason to remove it?

Reply:
There is a problem with the type then because it types as `string` should it be `string[]`